pkgs/development/compilers/teyjus/default.nix

```diff
@@ -1,22 +1,33 @@
-{ lib, stdenv, fetchFromGitHub, omake, ocaml, flex, bison }:
+{ lib, stdenv, fetchFromGitHub, fetchpatch, omake, ocaml, flex, bison }:
```

Comment:
`fetchpatch` is no longer needed if we bump to an unstable release instead of just grabbing the relevant commits as patches.

Reply:
Done. Thanks!